main.go

```diff
@@ -65,11 +66,13 @@ func main() {
 		exit("Failed to fetch incidents from Datadog: %v", err)
 	}
 
-	pagerdutyTeam := *team
+	var pages []*page
 	if pdTeam != nil {
-		pagerdutyTeam = *pdTeam
+		*pdTeam = strings.ToLower(*pdTeam)
+		pages, err = fetchPages(*pdTeam, *since, *until, *team)
+	} else {
+		pages, err = fetchPages(*team, *since, *until, "")
```

Comment:
What I thought could be more generic - still keep the team, but add a tags filter (as an additional parameter), that way you can still prefilter by the team. Concern is with having to filter a lot of pages.

Reply:
Makes sense, I added an additional tags filter and removed these changes